### DISCUSSION TITLE
Passive skills and named effects

### DISCUSSION THREAD

**Discussion by AdvTimber** [misthero/dnd5e-custom-skills#38] [Q&A]
Ola.

I'm trying to use this to remove investigation and perception and combine them into a new skill.

Awareness(INT)

I got it all setup, but now, using monk token bar, see passive awareness instead of passive investigation and perception.

How do I go about finding out what the active effect naming convention is for the new skill.

Instead of skills.prc.passive?

**Comment by AdvTimber** [comment]
Is the skill active effect "cus_0"?

**Comment by misthero** [comment]
probably yes, you should be able to see all the custom skills (cus_X) and custom abilities (cua_X) in the setting page between parentesis.

![immagine](https://github.com/misthero/dnd5e-custom-skills/assets/3662610/0af20fa8-e2e7-4b3d-9761-896be4805157)
